### PR TITLE
Add Android fixes

### PR DESCRIPTION
**Pull request by vishal-elear**

### CODE REVIEW

**Review by elear-solutions-dev — APPROVED** [review]

File: configure.ac [hunk 1]
```diff
@@ -165,7 +165,7 @@ AS_IF([test "x$enable_Bsymbolic" = "xcheck"],
 AS_IF([test "x$enable_Bsymbolic" = "xyes"], [JSON_BSYMBOLIC_LDFLAGS=-Wl[,]-Bsymbolic-functions])
 AC_SUBST(JSON_BSYMBOLIC_LDFLAGS)
 
-AX_APPEND_COMPILE_FLAGS([-Wall -Werror -Wcast-qual -Wno-error=deprecated-declarations])
+AX_APPEND_COMPILE_FLAGS([-Wall -Werror -Wcast-qual -Wno-error=deprecated-declarations -fPIC])
```
Comment (elear-solutions-dev) [inline configure.ac]: instead move -fPIC to aconfigure.sh - using CFLAGS input to the ./configure statement?

That way you don't change anything in the original build files.

**Review by elear-solutions-dev — CHANGES_REQUESTED** [review]

File: configure.ac (hunk 1 above)
Comment (vishal-elear) [reply configure.ac]: Wouldn't that still cause issues while building a Linux static library for COCO SDK or any other library using this as a dependency?

Comment (elear-solutions-dev) [reply configure.ac]: We can add the same on the external lconfigure.sh build script.

In general its best not to change 3rd party code as far as possible. It allows us to upgrade easily without remembering to "port" our changes across various versions.

**Review by elear-solutions-dev — APPROVED** [review]
please squash your commit messages

**Pull request merged**